components/ReportList.vue

```diff
@@ -9,10 +9,12 @@ import {
 import { useSingleReport, useCommit } from "../composables";
 import { BusinessButton, ShieldIcon } from "@axodotdev/fringe/lib";
 import { getVersionChangeText } from "../utils/versions";
+import { AxoModal } from "@axodotdev/fringe/lib";
```

Comment:
**todo:** Move this into the `@axodotdev/fringe` import above. (Not sure why the linter did not catch this one.)

Reply:
done